### PR TITLE
creating a view and controller

### PR DESCRIPTION
**Pull request by f-medina**
## Summary
- First Controller and view
## Trello Card
https://trello.com/c/SwF8ZkMh/17-crear-un-controller-y-una-vista-de-bienvenida

### CODE REVIEW

File: src/main/java/wolox/training/controllers/GreetingController.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class GreetingController {
+
+    @GetMapping("/greeting")
+    public String greeting(@RequestParam(name="name", required=false, defaultValue="World") String name, @NotNull Model model) {
```
Comment (torsello) [inline src/main/java/wolox/training/controllers/GreetingController.java]: In this case, you are using 'name' and 'greeting' more than one time.

It's always a good practice to extract this strings on a constant, to reuse it.

**Review by torsello — CHANGES_REQUESTED** [review]
Some comments :)

**Review by AleFalcon — APPROVED** [review]
With the comment from Matias, it's ok this PR.

Good job.

File: src/main/java/wolox/training/controllers/GreetingController.java [hunk 2]
```diff
@@ -7,11 +7,14 @@
...
 @Controller
+
 public class GreetingController {
 
+    private static final String NAME = "name";
+
     @GetMapping("/greeting")
-    public String greeting(@RequestParam(name="name", required=false, defaultValue="World") String name, @NotNull Model model) {
-        model.addAttribute("name", name);
+    public String greeting(@RequestParam(name=NAME, required=false, defaultValue="World") String name, @NotNull Model model) {
```
Comment (torsello) [inline src/main/java/wolox/training/controllers/GreetingController.java]: You must extract "World" and "greeting" on constants too :)

**Review by torsello — APPROVED** [review]
LGTM!

**Pull request merged**